Makefile

```diff
@@ -27,12 +27,18 @@ release:
 	GOOS=windows GOARCH=amd64 go build -o ./bin/${BINARY}_${VERSION}_windows_amd64
 
 install: build
-	mkdir -p ~/.terraform.d/plugins/${HOSTNAME}/${NAMESPACE}/${NAME}
-	mv ${BINARY} ~/.terraform.d/plugins/${HOSTNAME}/${NAMESPACE}/${NAME}
+	mkdir -p ~/.terraform.d/plugins/${HOSTNAME}/${NAMESPACE}/${NAME}/${VERSION}/${OS_ARCH}
+	mv ${BINARY} ~/.terraform.d/plugins/${HOSTNAME}/${NAMESPACE}/${NAME}/${VERSION}/${OS_ARCH}
 
 test:
 	go test -i $(TEST) || exit 1
 	echo $(TEST) | xargs -t -n4 go test $(TESTARGS) -timeout=30s -parallel=4
 
 testacc:
-	TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout 120m
\ No newline at end of file
+	TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout 120m
+
+docs:
+	tfplugindocs generate
+
+.PHONY: test docs
```

Comment:
Please explain this for me

Reply:
I mean .PHONY

Reply:
A phony target is one that is not really the name of a file or directory; rather it is just a name for a recipe to be executed when you make an explicit request.

since both test & docs are existing directories,  if we don't mark them as phony targets when you run `make {test|docs}` you will get `{target} is up to date` and nothing will be done, phony fixes that.